site/genref/config.yaml

```diff
@@ -0,0 +1,15 @@
+hiddenMemberFields:
+  - "TypeMeta"
+  - "ObjectMeta"
+
+apis:
+  - name: kueue
+    title: Kueue API
+    package: ../..
+    path: apis/kueue/v1beta1
```

Comment:
Is it possible to add a description that can be shown in the reference page?

https://deploy-preview-1263--kubernetes-sigs-kueue.netlify.app/docs/reference/

Reply:
Yes, from the templates. Done.